static/js/components/Footer.tsx

```diff
@@ -5,7 +5,7 @@ export default function Footer(): JSX.Element {
     <footer>
       <div className="p-4">
         <div className="float-left">
-          &copy;2021 Massachusetts Institute of Technology
+          &copy;2021-{RELEASE_YEAR} Massachusetts Institute of Technology
```

Comment:
My inner typographer is requesting this change:

```suggestion
          &copy;2021&ndash;{RELEASE_YEAR} Massachusetts Institute of Technology
```

https://en.wikipedia.org/wiki/Dash#Ranges_of_values